elated-egrets/bangazon-internal-site
# Pwm employee edit

## DESCRIPTION
Edit employee information
go to list of employees
click on employee
click edit employee
change info then click edit on far right --edit
## How to pull down
Commands

```
git fetch --all
git checkout pwm-employee-edit
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
2. run command: django_data.sh employee_portal faker_factory
3. run command: python manage.py test

## What you changed
	modified:   employee_portal/forms/__init__.py
	modified:   employee_portal/forms/employee_add_form.py
	modified:   employee_portal/models/employee_model.py
	modified:   employee_portal/templates/employee_portal/employee_detail.html
	deleted:    employee_portal/templates/employee_portal/employee_list.html
	modified:   employee_portal/urls.py
	modified:   employee_portal/views/__init__.py
	modified:   employee_portal/views/employee_list_view.py


	employee_portal/forms/employee_edit_form.py
	employee_portal/templates/employee_portal/employee_edit.html
	employee_portal/tests/test_employee_edit.py
	employee_portal/views/employee_edit_view.py

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
